test/components/executions/execution-status.js

```diff
@@ -1,15 +1,53 @@
+////*************************************************************************************************************************************
```

Comment:
We should probably attempt to include this information in our commit/PR log and not attempt to maintain in-line comments re: edit reasons.    At a minimum, this warrants a team norms conversation.

Reply:
These were my comments as part of an effort to begin some sort of test documentation. I found a lack of documentation on what the test was trying to accomplish, as well as how to even call it to run the test script file. I agree we should have a conversation on where this type of information can be stored/shared. Thanks @Jkovarik.